Long-Term SQL Backup retention policy name

PeterSelchDahl opened the pull request:
Long-Term SQL Backup retention policy name. See case "RE: [REG:117121817338808]  Long-Term SQL Backup  Initial Response" for more information

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

PeterSelchDahl committed:
Long-Term SQL Backup retention policy name

Long-Term SQL Backup retention policy name. See case "RE: [REG:117121817338808]  Long-Term SQL Backup  Initial Response" for more information

GitHubber17 committed:
Update sql-database-long-term-backup-retention-configure.md
